src/v/cluster/config_frontend.cc

```diff
@@ -15,6 +15,7 @@
 #include "cluster/controller_service.h"
```

Comment:
Test suggestion (though feel free to test some other way), it'd be nice to have a fixture test to use the config frontend, and then validate the batches in the log by reading the controller log records. Here's an example of a test that does something similar with cluster recovery controller batches https://github.com/redpanda-data/redpanda/blob/ec9cf50926b5c666a61937a33007f602f0c4c91c/src/v/cluster/cloud_metadata/tests/offsets_recovery_test.cc#L826